packages/checkout/widgets-lib/src/lib/balance.ts

```diff
@@ -59,6 +61,15 @@ export const getAllowedBalances = async ({
     type: allowTokenListType,
   });
 
+  // Map token icon assets
+  allowList.tokens = allowList.tokens.map((token) => ({
+    ...token,
+    icon: getTokenImageByAddress(
+      checkout.config.environment as Environment,
+      isNativeToken(token.address) ? token.symbol.toLowerCase() : token.address ?? '',
```

Comment:
nit: the toLowerCase() here is unneccessary as it will happen in the function itself

Reply:
In this case I'm just making sure because its mapping to the svg URL.. which I guess shouldn't be case sensative.. but the imx.svg and eth.svg are lowercase and prefer to map that exactly.

Reply:
Ahh cool .. you are correct! sorry missed that deeper in.. taken this out. Nice catch!